rfcs/0050-vm-syscalls-3/0050-vm-syscalls-3.md

```diff
@@ -213,5 +186,240 @@ This syscall might return the following errors:
 
 In case of errors, `addr` and `index` will not contain meaningful data to use.
 
+## Error Code
+
+Five new error types added:
+
+- Error code 5: The file descriptor is invalid during syscall [Wait].
+- Error code 6: The file descriptor is not owned by this process.
+- Error code 7: The other end of the pipe is closed.
+- Error code 8: The maximum count of spawned processes has been reached.
+- Error code 9: The maximum count of created pipes has been reached.
+
+It's possible for read/write/wait operations to wait for each other, leading to a deadlock state. In such cases, CKB-VM would throw an internal error and terminate immediately.
+
+## Cycles
```

Comment:
When a VM gets suspended (and maybe later resumed), will it consume cycles? Can we document them here as well?